apps/builder/app/canvas/features/webstudio-component/selected-instance-connector.ts

```diff
@@ -93,7 +93,7 @@ export const SelectedInstanceConnector = ({
     setDataCollapsed(instance.id, true);
 
     const element = instanceElementRef.current;
-    if (element === undefined) {
+    if (element == null) {
```

Comment:
if you use null, then we can go with strict equal

Reply:
checked again, after the change to initial null, it's always either null or html element, so non-strict check is not required

I personally don't care about this, since it's not doing any harm here, but it goes against the rfc and I don't want to make it confusing for other readers by making it inconsistent